src/main/java/com/prgrms/springbasic/domain/customer/controller/CustomerApiController.java

```diff
@@ -0,0 +1,35 @@
+package com.prgrms.springbasic.domain.customer.controller;
+
+import com.prgrms.springbasic.domain.customer.dto.CreateCustomerRequest;
+import com.prgrms.springbasic.domain.customer.dto.CustomerResponse;
+import com.prgrms.springbasic.domain.customer.service.CustomerService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/customers")
```

Comment:
버전도 명시해주면 좋을 것 같습니다!
```suggestion
@RequestMapping("/api/v1/customers")
```